tools/pybind11Tools.cmake

```diff
@@ -124,10 +124,10 @@ endfunction()
 
 # Build a Python extension module:
 # pybind11_add_module(<name> [MODULE | SHARED] [EXCLUDE_FROM_ALL]
-#                     [NO_EXTRAS] [THIN_LTO] source1 [source2 ...])
+#                     [NO_EXTRAS] [THIN_LTO] [OPT_SIZE] source1 [source2 ...])
 #
 function(pybind11_add_module target_name)
-  set(options MODULE SHARED EXCLUDE_FROM_ALL NO_EXTRAS SYSTEM THIN_LTO)
+  set(options "MODULE;SHARED;EXCLUDE_FROM_ALL;NO_EXTRAS;SYSTEM;THIN_LTO;OPT_SIZE")
```

Comment:
I'm guessing the change in syntax was for cmake-format wanting to wrap? It's fine, just curious. This is more symmetric with the inline def in the other module.

Reply:
Yep, saw this when I rebased.

Reply:
Indeed -- pre-commit rewrote this into a bunch of lines, and I just set it to follow the convention in the other file.